### DISCUSSION TITLE
i cant compile

### DISCUSSION THREAD

**Discussion by FurryColtin** [FunkinCrew/Funkin#1932] [Show and tell]
its saying that you need to and mark this "go check out https://github.com/FurryColtin/FNF-Psych-Engine-Optinal-Features" and so i did and i gotta say it has a built in music player epik

also this post is a troll but its true it does

**Comment by ghost** [comment]
Kill yourself

**Comment by tposejank** [comment]
self advertisement moment

**Comment by libauror-a** [comment]
https://user-images.githubusercontent.com/75984453/149653653-0e1133bc-a101-4a5b-b3a4-cc1243c426fa.mp4

**Comment by stilic** [comment]
its not for psych engine but for the original fnf dumbass

**Comment by libauror-a** [comment]
![image](https://user-images.githubusercontent.com/75984453/150636025-7896ff52-4e8c-42e8-87d4-7e34e108d83a.png)

**Comment by ghost** [comment]
Bruh. That video is for psych engine, not the base game

**Reply by ghost** [reply]
Also idgaf if this is supposed to be a troll
